### ISSUE TITLE
Is there a Save As option in Vimium

### ISSUE THREAD

**Issue by alphaCTzo7G** [philc/vimium#2774]
I noticed this PR from a while back which mentioned that it is not possible to do save as in VIMIUM, but I dont see that option in help menu or otherwise in the github wiki.

How do I save videos or images to my desktop with VIMIUM?

https://github.com/philc/vimium/issues/331

**Comment by smblott-github** [comment]
No, there isn't.  And there has been much discussion about how to extend Vimium's capabilities without bloating the UI to the point of unusability.
